### ISSUE TITLE
Xcode 13: Could not find 'SupportedLanguage' enum within 'BartyCrouch' enum within path

### ISSUE THREAD

**Issue by Sam-Spencer** [Flinesoft/BartyCrouch#235]
## Expected Behavior
BartyCrouch setup and installation for a project which worked on Xcode 12 should behave the same (or at least without warnings / errors) on Xcode 13.

## Actual Behavior
When opening a project in Xcode 13 with BartyCrouch and running `update` then `lint`, a warning appears in Xcode:

```bash
Could not find 'SupportedLanguage' enum within 'BartyCrouch' enum within path.
```

I have changed nothing in my BartyCrouch setup, `.toml` file, or with my localizations. Not seeing this warning in Xcode ≤ 12.

## Steps to Reproduce the Problem

  1. Open a working project with a BartyCrouch run script in the Build Phases in Xcode 13
  2. Build the project
  3. A warning is thrown

## Specifications

  - Version: 4.7.1
  - Platform: macOS Big Sur 11.6
  - IDE Version: Xcode 13.0 (13A233)

**Comment by khorbushko** [comment]
+1

**Comment by Hengyu** [comment]
This [PR](https://github.com/Flinesoft/BartyCrouch/pull/234) updated the `SwiftSyntax` dependency which could solve the problem you faced.

**Comment by Jeehut** [comment]
True, this should be resolved by #234 and the release of version `4.8.0`, please run `brew upgrade bartycrouch`.
